app/presenters/two_factor_options_presenter.rb

```diff
@@ -68,6 +68,30 @@ def phone_options
     [TwoFactorAuthentication::PhoneSelectionPresenter.new]
   end
 
+  def sms_option
+    if piv_cac_required? ||
+       aal3_only? ||
+       IdentityConfig.store.hide_phone_mfa_signup ||
+       phone_options.any?
+
+       return []
```

Comment:
1. Minor spacing issues (remove blank line, extra space before `return`.
2. The `return` isn't strictly necessary here, since it'd be returned implicitly

```suggestion
      []
```